components/SkeletonCard.vue

```diff
@@ -22,6 +22,11 @@ export default Vue.extend({
     noText: {
       type: Boolean,
       default: false
+    },
+    cardShape: {
+      type: String,
+      required: false,
```

Comment:
You don't technically need this, unless you found away to make properties required by default?

Reply:
Ah I didn't realise this. And no we haven't, AFAIK.